packages/dds/merge-tree/src/segmentGroupCollection.ts

```diff
@@ -44,9 +44,17 @@ export class SegmentGroupCollection {
     }
 
     public copyTo(segment: ISegment) {
-        walkList(
-            this.segmentGroups,
-            (sg) => segment.segmentGroups.enqueue(sg.data),
-        );
+        this.segmentGroups.walk((sg) => segment.segmentGroups.enqueueOnCopy(sg, this.segment));
+    }
+
```

Comment:
hey @scarlettjlee, there are some build failures from this commit: https://github.com/microsoft/FluidFramework/pull/11961/files#diff-15ecf9660a7728b07fbe68b0c42da3091aff3bf818ef5236c81a5f10bc21c606. Can you please advise on the right fix here?